webapp/channels/src/utils/url.tsx

```diff
@@ -168,6 +167,24 @@ export function validateChannelUrl(url: string, intl?: IntlShape): Array<React.R
     return errors;
 }
 
+// Returns true when the URL should not trigger any external requests.
+// Currently returns true only for permalinks
+export function doesNotTriggerExternalRequests(url: string, siteURL?: string): boolean {
```

Comment:
We might want to flip this to be something like `canTriggersExternalRequests` since I noticed you call this as `!doesNotTriggerExternalRequests` currently. Flipping it around would avoid that double negative.

Reply:
Done. It kinda makes the function a little less clear, but it removes the double negative, so win?